### ISSUE TITLE
Designer does not send the Grid on/off status to students.

### ISSUE THREAD

**Issue by cbradyatinquire** [MathNetDev/MathNetClient#27]
As part of the effort to make the Activity Designer more of a WYSIWYG interface, we would like for the grid settings to be sent to students on pressing Send View To All.

**Comment by IsaacNPsmash** [comment]
The designer does not send the axis setting in any way. I changed the axis to polar coordinates and clicked Send View To All to no effect.

**Comment by ghost** [comment]
This issue has been fixed for handling the grid visibility. The one for the axes is referenced in another issue and is not working for the 3D graphics view as expected due to an existing issue. Waiting for clarification from the Geogebra team for that issue. But grid visibility should be fixed on test for all views. Please test and let me know. Thanks.

**Comment by IsaacNPsmash** [comment]
Tested 10/17/18 @ 10:03 am cst Test branch
Passed with conditions
I tested this sending all the different grid visibilities to three students in the same group. I varied yes-no the axes and the different grids (coordinate grid, polar grid, triangle grid). All went to students well. The only time it did not work was upon entry into a group. When a student enters into a group, the default is axes without grid regardless of what the teacher sent. I believe this is a part of a larger issue of student-late-entry (connected to issue 34 about default perspectives).
